DNN Platform/Library/Entities/Users/UserInfo.cs

```diff
@@ -114,6 +114,36 @@ public string FirstName
         [Browsable(false)]
         public bool IsSuperUser { get; set; }
 
+
+        /// <summary>
+        /// Gets whether the user is in the portal's administrators role
+        /// </summary>
+        private bool? isAdmin;
```

Comment:
Given that the calculated value is never stored/used, no need for this to be a private backer

Reply:
Not sure I am on the same page, but I am pretty confident that once we have this value then extension developers will begin using this value more and more instead of coding their own IsAdmin logic. And that might require accessing the value multiple times in your code.

On the other hand one could argue that maybe an extension developer might change the user's roles and then recalculation would be necessary.

On the fence about it. Given that the calculation is not trivial I persisted the value.